syft_tensorflow/serde/serde.py

```diff
@@ -72,9 +72,63 @@ def _detail_tf_tensor(worker, tensor_tuple) -> tf.Tensor:
     return tensor
 
 
+def _simplify_tf_tensorshape(tensorshape: tf.TensorShape) -> bin:
+    """
+    This function converts a TF tensor shape into a serialized list.
+
+    Args:
+      tensor (tf.TensorShape): an input tensor shape to be serialized
+
+    Returns:
+      tuple: serialized tuple of TF tensor shape.The first value is
+      the binary for the TensorShape object. The third is the
+      chain of abstractions.
+    """
+
+    tensorshape_list_ser = syft.serde.serde._simplify(
+        tensorshape.as_list())
+
+    chain = None
+    if hasattr(tensorshape, "child"):
```

Comment:
will this ever be true? I don't think we've hooked TensorShape. that might also be a good thing to do, since judging from its methods it seems to have quite a few responsibilities